.github/workflows/test.yml

```diff
@@ -77,7 +77,7 @@ jobs:
 
   test-integration-rootless:
     runs-on: ubuntu-20.04
-    timeout-minutes: 60
+    timeout-minutes: 30
```

Comment:
Probably you accidentally made this change

Reply:
yep, fixed

Reply:
fixed.